src/components/access_group/AccessGroupMembersEdit.vue

```diff
@@ -200,8 +207,7 @@ export default {
     RadioSelect,
     Select,
   },
-  props: [],
-  mounted() {},
+  mixins: [MembersListMixin],
```

Comment:
I just saw this file exports the wrong name `AccessGroupInformationEdit` instead of `AccessGroupMembersEdit`

Reply:
👀 ...nothing to see here folks...